PoohQuotes/app/src/main/java/com/example/poohquotes/Quotes.java

```diff
@@ -0,0 +1,47 @@
+package com.example.poohquotes;
+
+/**
+ * The Quotes class stores the quotes in a private array
+ * the public method getQuote returns the next quote in the array
+ * if all quotes from the array were sent, starts from the beginning
+ *
+ */
+
+public class Quotes {
+    private static String[] myQuotes =
```

Comment:
Why not using a class "Quote" with the text & the author and using a Quote array? it is much more OOP oriented.